src/services/lease-service/adapters/xpand/tenant-lease-adapter.ts

```diff
@@ -377,12 +377,11 @@ const getContactQuery = () => {
       'cmadr.adress3 as postalCode',
       'cmadr.adress4 as city',
       'cmeml.cmemlben as emailAddress',
-      'cmobj.keycmobj as keycmobj',
+      'cmctc.keycmobj as keycmobj',
       'cmctc.keycmctc as contactKey'
     )
-    .innerJoin('cmobj', 'cmobj.keycmobj', 'cmctc.keycmobj')
-    .leftJoin('cmadr', 'cmadr.keycode', 'cmobj.keycmobj')
-    .leftJoin('cmeml', 'cmeml.keycmobj', 'cmobj.keycmobj')
+    .leftJoin('cmadr', 'cmadr.keycode', 'cmctc.keycmobj')
+    .leftJoin('cmeml', 'cmeml.keycmobj', 'cmctc.keycmobj')
```

Comment:
Vad innebär de här joinändringarna?

Reply:
Förut gjordes en join mot cmobj men som synes i select så behövs inte det eftersom vi inte använder någon data från tabellen utan endast nyckeln. Och för att joina in cmadr och cmeml så kan vi använda sekundärnyckeln i cmctc istället 

Anledningen till ändringen är att jag hittade ett case med en användare som inte kunde läsas alls eftersom vi gjorde en inner join på cmobj och den användaren saknade rad där. Vi hade förstås kunnat göra en left istället och det hade ju funkat men faktum kvarstår att vi inte behöver joina mot den alls eftersom vi inte använder datan därifrån vad jag kan se...

Reply:
Alright 👍